xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -199,6 +199,58 @@
           </listitem>
         </itemizedlist>
 
+        <para>In addition, the Pull request events contains a field called
+        action. OBS supports a different subset of Pull request event actions,
+        depending on the SCM. For GitHub and Gitea, the following set of actions
+        is supported:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>closed</para>
+          </listitem>
+
+          <listitem>
+            <para>opened</para>
+          </listitem>
+
+          <listitem>
+            <para>reopened</para>
+          </listitem>
+
+          <listitem>
+            <para>synchronize</para>
+          </listitem>
+
+          <listitem>
+            <para>synchronized</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>For Gitlab's Merge request events, the following actions are
+        supported:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>close</para>
+          </listitem>
+
+          <listitem>
+            <para>merge</para>
+          </listitem>
+
+          <listitem>
+            <para>open</para>
+          </listitem>
+
+          <listitem>
+            <para>reopen</para>
+          </listitem>
+
+          <listitem>
+            <para>update</para>
+          </listitem>
+        </itemizedlist>
+
         <para>Other SCMs like GitLab have similar behaviour; the main
```

Comment:
The paragraph which starts in this line should be moved right after the events list. 
Take into account we only list GitHub events and use this paragraph to invite users to discover the events from other SCMs.

Unlike the actions, that we list all of them (GitHub, GitLab, Gitea).

Reply:
An alternative would be listing all the possible events for all the SCMs. And then rephrase that paragraph to simply invite the users to read more (and don't move it).